radix-engine-common/src/data/scrypto/custom_well_known_types.rs

```diff
@@ -443,7 +443,7 @@ create_well_known_lookup!(
             ROLE_ASSIGNMENT_TYPES_START + 0,
             named_enum(
                 // This is a clearer name for UIs, but not time to change in the code right now
-                "Rule",
+                "AccessRule",
```

Comment:
@dhedey Needed to make changes to this file since code generation relies on the name in the metadata to match the name of the Rust type.

Reply:
Then we should change codegen IMO to have an override for native type names

Reply:
Just reminder to remove these as part of final markups :)